### PR TITLE
(maint) Prepare for Puppet 6

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -125,7 +125,7 @@ def device_model
         apply_manifest('include cisco_ios')
         on host, puppet('plugin', 'download', '--server', host.to_s)
         on host, puppet('device', '-v', '--waitforcert', '0', '--user', 'root', '--server', host.to_s), acceptable_exit_codes: [0, 1]
-        on host, puppet('cert', 'sign', '--all'), acceptable_exit_codes: [0, 24]
+        on host, 'puppetserver ca sign --all', acceptable_exit_codes: [0, 24]
```
Comment (justinstoller) [inline spec/spec_helper_acceptance.rb]: So, depending on the support matrix for this module we probably want to check for the module version and if less than 5.99 do the old way, if more do this way. Here's an example from beaker-puppet that would probably work for you: https://github.com/puppetlabs/beaker-puppet/pull/67/commits/59ec507a5e86bd52592c47cf3edefb650c71f2da#diff-1ad39dbbbad154ea718af70e36bca745R833

**Comment by justinstoller** [comment]
This is hopefully an example of what needs to change and how it can be changed for Puppet 6 support. I haven't had any success running module acceptance (on most of the modules) locally, yet. So I have to admit these changes are yet untested. Let me know if there's anything else I can do.

**Comment by davidmalloncares** [comment]
@willmeek is this something you could take a look at in the morning?

**Comment by willmeek** [comment]
@davidmalloncares Had a look earlier, minimal changes but was hoping to get @tphoney 's input as he would probably have more knowledge of the extent of Puppet 6 changes?

**Comment by willmeek** [comment]
Updated branch with:
- Update to README (@clairecadman could you review this?)
- Update to spec_helper_acceptance based on version of Puppet

**Comment by tphoney** [comment]
👍

**Pull request merged**